resources/migrations/000_migrations.yaml

```diff
@@ -9868,10 +9868,17 @@ databaseChangeLog:
             onDelete: SET NULL
 
   - changeSet:
-      id: v420.00-070
+      id: v42.00-070
       author: camsaul
       comment: >-
-        Added 0.42.0 - add Database.settings column to implement Database-local Settings
+        Added 0.42.0 - add Database.settings column to implement Database-local Settings. Renamed from v420.00-070
```

Comment:
```suggestion
        Added 0.42.0 - add Database.settings column to implement Database-local Settings. Renamed from v42.00-070
```

Reply:
This comment is correct. this migration used to be erroneously named `v420.00-070`. I renamed it, gave it a precondition and left a note about its weird old name

<img width="1029" alt="image" src="https://user-images.githubusercontent.com/6377293/149039570-be8f1f97-f9d7-4503-9bb1-72adea7ffdba.png">